### PR TITLE
Docs: Adding an example for the spread operator to prefer-spread.md (#7802)

### PR DESCRIPTION
**Pull request by butlermd**
**What is the purpose of this pull request? (put an "X" next to item)**

[ X ] Documentation update
[ ] Bug fix ([template](https://raw.githubusercontent.com/eslint/eslint/master/templates/bug-report.md))
[ ] New rule ([template](https://raw.githubusercontent.com/eslint/eslint/master/templates/rule-proposal.md))
[ ] Changes an existing rule ([template](https://raw.githubusercontent.com/eslint/eslint/master/templates/rule-change-proposal.md))
[ ] Add autofixing to a rule
[ ] Add a CLI option
[ ] Add something to the core
[ ] Other, please explain:

Adding an example of using the spread operator per [#7802](https://github.com/eslint/eslint/issues/7802). 

Do I need to squash the commits?

### CODE REVIEW

**Comment by vitorbal** [comment]
Thanks! No need to squash the commits, we can take care of that when we merge.

**Comment by butlermd** [comment]
Sounds good.

File: docs/rules/prefer-spread.md [hunk 1]
```diff
@@ -41,6 +41,11 @@ Examples of **correct** code for this rule:
 ```js
 /*eslint prefer-spread: "error"*/
 
+// Using the spread operator
+
```
Comment (vitorbal) [inline docs/rules/prefer-spread.md]: Minor nit: could you get rid of this empty line, as none of the other examples below have one?

**Comment by butlermd** [comment]
Yes, that makes sense. While I'm looking at it, the incorrect examples have an empty line between them while the correct examples don't. Should I also remove those empty lines (33 and 35)?

**Comment by platinumazure** [comment]
@butlermd Could you please [sign our Contributor License Agreement](https://cla.js.foundation/eslint/eslint?pullRequest=7804)? We can't accept pull requests unless the committer(s) sign the CLA. Thanks!

**Comment by butlermd** [comment]
I did and when I try to sign it again it says I've already signed. Some issue with the integration?

**Comment by platinumazure** [comment]
@butlermd Does the CLA signature match your Git name and e-mail address? Some or all of those have to match, otherwise your signature won't be associated with the commits.

Come by our [chat room](https://gitter.im/eslint/eslint) if you need help looking into that.

**Comment by not-an-aardvark** [comment]
It looks like the git commit author/email of your most recent commit "removed excess whitespace per vitorbal's comment" is different from the author/email of the other commits. The bot requires that all the commit authors must have signed the CLA.

Assuming the last commit was also created by you, you could amend it to use the same author/email as before.

**Comment by butlermd** [comment]
Thanks @not-an-aardvark and @platinumazure. Got it figured out. Committed with a key that's associated with my work email. Added the email to my github profile. I think I'll need to make another commit to get the integration to recognize it. Should I make the change to the spacing of the incorrect examples to make it consistent with the correct examples?

**Comment by platinumazure** [comment]
@butlermd Note that adding another commit won't fix the CLA issue by itself, because you'll still have the old commit with the wrong name/e-mail. You need to amend that previous commit and force-push. Stop by in [our chat](https://gitter.im/eslint/eslint) if you need help with that.

**Comment by vitorbal** [comment]
@butlermd sure, makes sense to make sure they're all consistent!

**Comment by butlermd** [comment]
I amended the commit and force pushed. It appears to have fixed the license issue.

**Comment by vitorbal** [comment]
Thanks for the contribution, @butlermd!

**Comment by butlermd** [comment]
Glad to help!

**Pull request merged**